### DISCUSSION TITLE
Make it possible to switch between loaders

### DISCUSSION THREAD

**Discussion by leonard84** [FakeFishGames/Barotrauma#8432] [Ideas and suggestions]
It would be great if we could link two loaders to a Railgun, and switch between the active loader via components. This way we can quickly switch between normal and canister shells without having to manually transfer 5 shells. Of course this would apply to all other turrets with multiple ammo types as well.

**Comment by JosiahRose** [comment]
I know this post is old, but I stumbled onto it just after I built a solution to this exact problem. 

I made an item assembly with some cheeky wiring that allows you to switch ammo types with a button press.

Here's a link: https://steamcommunity.com/sharedfiles/filedetails/?id=2961242956
